device_id: 0x30
resolution: 18